plugins/MobileMessaging/ReportRenderer/Sms.php

```diff
@@ -73,15 +73,15 @@ public function renderReport($processedReport)
             'ColumnCallbackReplace',
             array(
                  array_merge(array_keys($multiSitesAPIMetrics), $evolutionMetrics),
-                 function ($value) use ($floatRegex) {
-                     return preg_replace_callback(
+                    function ($value) use ($floatRegex) {
```

Comment:
Move the `array_merge` one character left and this newly added indentation won't be needed.

Reply:
Plus can do the shorthand array once at it.

Reply:
done